src/main/java/org/b612foundation/adam/opm/OrbitParameterMessage.java

```diff
@@ -163,7 +174,7 @@ public boolean equals(Object obj) {
     // @formatter:off
     return Objects.equals(adamFields, other.adamFields)
         && Objects.equals(ccsdsOpmVers, other.ccsdsOpmVers)
-        && Objects.equals(covariance, other.covariance)
+        && Objects.equals(cartesianCovariance, other.cartesianCovariance)
         && Objects.equals(header, other.header)
         && Objects.equals(keplerian, other.keplerian)
```

Comment:
should this be adjusted to reflect the addition of the new fields?

Reply:
Ah yes. I'm thinking I'm going to replace the equals and hash with the Lombok one to avoid this problem...

Reply:
I think that's a great idea!